sequencer/src/header.rs

```diff
@@ -683,7 +695,11 @@ mod test_headers {
         // The current fake implementation of fetch_fee_receipts returns
         // some fee info. To validate the proposal we need to insert these
         // records here.
-        for fee_info in fetch_fee_receipts(None, None) {
+        for fee_info in genesis_state
```

Comment:
Comment above can be removed.

Reply:
fixed b4c8b422f65ba507fb7bc50639cac03e4904c88b